yamcs-core/src/main/java/org/yamcs/tctm/ccsds/UdpTmFrameLink.java

```diff
@@ -115,4 +168,19 @@ protected void doEnable() throws SocketException {
     protected Status connectionStatus() {
         return Status.OK;
     }
+
+    /**
+     *  A parsing method used to compare the frame first words and the ASM as strings
+     */
+    public static String convertBytesToHexadecimal(byte[] byteArray)
```

Comment:
function not needed if Arrays.equals is used to compare binary but note that we have a conversion to hexadecimal in org.yamcs.utils.StringConverters class